### PR TITLE
helm charts for snapshot-controller

### PR DESCRIPTION
**Pull request by min-zh**

### CODE REVIEW

File: src/test/snapshot-controller/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+
+appVersion: 2.1.1
+name: snapshot-controller
+description: A Helm chart for snapshot-controller
+version: 2.1.1
+kubeVersion: ">=1.17.0-0"
+home: https://github.com/kubernetes-csi/external-snapshotter
+sources:
+  - https://github.com/kubernetes-csi/external-snapshotter
```
Comment (hlwanghl) [inline src/test/snapshot-controller/Chart.yaml:10]: Should be this repo, i.e. https://github.com/kubesphere/helm-charts

File: src/test/snapshot-controller/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+
+appVersion: 2.1.1
+name: snapshot-controller
+description: A Helm chart for snapshot-controller
+version: 2.1.1
```
Comment (hlwanghl) [inline src/test/snapshot-controller/Chart.yaml:6]: `version` is not `appVersion`, and we usually start with something like `0.1.0`, in which `0` means it's unstable. When the chart becomes stable enough, we can promote it to `1.0.0`.

Comment (min-zh) [reply src/test/snapshot-controller/Chart.yaml]: OK

File: src/test/snapshot-controller/Chart.yaml (hunk 1 above)
Comment (min-zh) [reply src/test/snapshot-controller/Chart.yaml]: OK

**Pull request merged**